.github/workflows/i18n-clean-up.yml

```diff
@@ -0,0 +1,18 @@
+name: Clean up after Crowdin download
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
I think we should limit the branch here as well, to prevent this action from running at all on uninterested branches.

Should be able to do something like

```yaml
on:
  pull_request:
    types:
      - closed
    branches:
      - 'i18n_crowdin_translations'

```

Reply:
I just tried the change in the test repo and the job won't run if we limit the branch like this. 😕

Reply:
do we _need_ to clean up the branch, or will the `i18n-crowdin-download` action just overwrite everything on that branch anyway?

if it does need cleaning up, i agree with josh that we need to scope it to only run when the crowdin PR is closed. otherwise 99% of the time this action runs it would be pointless and do nothing 😅

Reply:
From what I've seen in the past, the download integration will just commit to the existing branch, rather than overwriting whatever's there. Which means over time it can get drift and cause issues, conflicts, etc.

I think it might be a good idea to just not have this action immediately, and if this becomes a problem again we can iterate and find a solution to it.

Reply:
I'll remove the action for now and we can keep it in the back of our minds in case we are going to need it.